pkg/compose/pull.go

```diff
@@ -286,23 +286,19 @@ func (s *composeService) pullRequiredImages(ctx context.Context, project *types.
 			i, service := i, service
 			eg.Go(func() error {
 				id, err := s.pullServiceImage(ctx, service, info, s.configFile(), w, quietPull, project.Environment["DOCKER_DEFAULT_PLATFORM"])
-				pulledImages[i] = id
```

Comment:
I think this should still be there - otherwise, `pulledImages` will always be full of empty strings, so we won't end up updating `images`

Reply:
Indeed, unexpected deletion
Seems my IDE is getting some fun with my nerves those days